### DISCUSSION TITLE
Week 4: Likes & Dislikes and Project ideas

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#98] [General]
**Likes:** I struggled at the beginning of the registration form. I like how the challenge is broken down into smaller steps which make it easier to learn. I like how it is interactive. I like how it gives feedback to me, which sometimes helps and sometimes doesn’t. 
**Dislikes:** I found the registration form to be challenging because I am a beginner and I still struggle understanding HTML and CSS. It took me a long time to figure it out. It doesn’t always explain why things are done in a particular way, which made it difficult for me to understand the concept. 
**Game Tried:**
**Elevator Saga:** This took me a long time. I am really struggling with coding. It really tested my programming skills and algorithms, which I am not great at. It seemed that there were more than 1 way to get the correct answer, which was nice. I found the game to be very time consuming and not very rewarding. 
**Updated Project Ideas**
1- Portfolio website with links to past projects to emphasize my degree. 
2- Online game with multiple levels and instructions. Each level will have a different, more challenging aspect to it. 
3- School Lunch Count website with checkboxes and input boxes to do daily lunch counts and totals online. 
4- Online interactive story where user needs to click on different aspects in the story to move forward. Instructions would be on the screen.

**Comment by gabebott** [comment]
The online interactive story sounds really interesting. That would be a really cool project!

**Comment by audreyhki** [comment]
Kassy, I understand your struggles and concerns with coding and feel the same way. The weekly tasks so far for this course have required much more of my time and attention than I would like. To stay positive, even though I'm not currently understanding everything, I tell myself that the repeated exposure and work will eventually stick. Although it may not make sense now, even if you have to "cheat" with a YouTube tutorial or Google search, you completing the task is aiding in your future understanding.

**Reply by metasean** [reply]
Very well said!
